src/resources/http.ts

```diff
@@ -16,12 +16,12 @@ import portfinder from 'portfinder';
 import { HttpClient } from '@nitric/proto/http/v1/http_grpc_pb';
 import { SERVICE_BIND } from '../constants';
 import * as grpc from '@grpc/grpc-js';
-import { HttpProxyRequest } from '@nitric/proto/http/v1/http_pb';
-import { HttpContext } from '../context/http';
+import { ClientMessage, HttpProxyRequest } from '@nitric/proto/http/v1/http_pb';
+import * as nodeHttp from 'http';
```

Comment:
Can we import the Server type from this instead?

Reply:
done